react-props-and-expressions/justin-app/src/App.js

```diff
@@ -0,0 +1,24 @@
+import logo from './logo.svg';
+import './App.css';
+import './CustomButton.css';
```

Comment:
Usually this should be imported in `CustomButton.js` to keep the CSS and JSX together. It's easier to maintain that way.